resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -231,6 +238,56 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class AlwaysShowScrollbars {
+    [DscProperty(Key)] [BinarySettingState] $ShowScrollBars = [BinarySettingState]::KeepCurrentValue
+
+    hidden [string] $DynamicScrollbarsProperty = 'DynamicScrollbars'
+
+    [AlwaysShowScrollbars] Get() {
+        $currentState = [AlwaysShowScrollbars]::new()
+        
+		if (-not(DoesRegistryKeyPropertyExist -Path $global:AlwaysShowScrollbarsRegistryPath -Name $this.DynamicScrollbarsProperty)) {
+            $ShowBars = [BinarySettingState]::Disabled                

```

Comment:
What is ShowBars? Did you mean `$currentState.AlwaysShow = [Status]::Disabled`?

Reply:
`ShowBars` is the intermediate variable needed to translate the binary Registry data into the textual `enum`  values that were requested.